ComfortZoneApp/app/src/main/java/com/example/comfortzone/fragments/FlightFragment.java

```diff
@@ -222,9 +217,31 @@ private void displayToggleListener() {
         tgCityDisplay.addOnButtonCheckedListener(new MaterialButtonToggleGroup.OnButtonCheckedListener() {
             @Override
             public void onButtonChecked(MaterialButtonToggleGroup group, int checkedId, boolean isChecked) {
+                if (checkedId == R.id.btnList && isChecked) {
+                    goToListView();
+                } else if (checkedId == R.id.btnMap && isChecked) {
+                    goToMapView();
+                }
             }
         });
     }
 
+    private void goToListView() {
+        Fragment fragment = CityListViewFragment.newInstance(cityList);
+        FragmentManager fragmentManager = getActivity().getSupportFragmentManager();
+        fragmentManager.beginTransaction()
+                .replace(R.id.flViewsContainer, fragment)
+                .addToBackStack(null)
+                .commit();
+    }
+
+    private void goToMapView() {
```

Comment:
the only difference between these 2 methods is the fragments, could you create a general method instead that takes in a fragment to transition?